### PR TITLE
Feature fix local compilers and make default

### PR DESCRIPTION
**Pull request by VoR0220**
* fixes a bug that destroys local compilers in release
* fixes default amounts sent so as not to completely destroy everyone's currently working solidity code
* includes a logging change from another PR.

### CODE REVIEW

File: cmd/pkgs.go [hunk 1]
```diff
@@ -52,10 +52,10 @@ func addPackagesFlags() {
 	packagesDo.Flags().StringVarP(&do.DefaultGas, "gas", "g", "1111111111", "default gas to use; can be overridden for any single job")
 	packagesDo.Flags().StringVarP(&do.Compiler, "compiler", "l", formCompilers(), "IP:PORT of compiler which Eris PM should use")
 	packagesDo.Flags().StringVarP(&do.DefaultAddr, "address", "a", "", "default address to use; operates the same way as the [account] job, only before the epm file is ran")
-	packagesDo.Flags().StringVarP(&do.DefaultFee, "fee", "n", "1234", "default fee to use")
+	packagesDo.Flags().StringVarP(&do.DefaultFee, "fee", "n", "9999", "default fee to use")
 	packagesDo.Flags().StringVarP(&do.DefaultAmount, "amount", "u", "9999", "default amount to use")
 	packagesDo.Flags().BoolVarP(&do.Overwrite, "overwrite", "t", true, "overwrite jobs of the same name")
-	packagesDo.Flags().BoolVarP(&do.LocalCompiler, "local-compiler", "z", false, "use a local compiler service; overwrites anything added to compilers flag")
+	packagesDo.Flags().BoolVarP(&do.RemoteCompiler, "remote-compiler", "z", false, "use a local compiler service; overwrites anything added to compilers flag")
```
Comment (zramsay) [inline cmd/pkgs.go]: um? the flag description no longer matches its functionality ...

Comment (VoR0220) [inline cmd/pkgs.go]: Yup. That needs fixing.

File: cmd/pkgs.go [hunk 2]
```diff
@@ -52,10 +52,10 @@ func addPackagesFlags() {
 	packagesDo.Flags().StringVarP(&do.DefaultGas, "gas", "g", "1111111111", "default gas to use; can be overridden for any single job")
 	packagesDo.Flags().StringVarP(&do.Compiler, "compiler", "l", formCompilers(), "IP:PORT of compiler which Eris PM should use")
 	packagesDo.Flags().StringVarP(&do.DefaultAddr, "address", "a", "", "default address to use; operates the same way as the [account] job, only before the epm file is ran")
-	packagesDo.Flags().StringVarP(&do.DefaultFee, "fee", "n", "1234", "default fee to use")
+	packagesDo.Flags().StringVarP(&do.DefaultFee, "fee", "n", "9999", "default fee to use")
 	packagesDo.Flags().StringVarP(&do.DefaultAmount, "amount", "u", "9999", "default amount to use")
 	packagesDo.Flags().BoolVarP(&do.Overwrite, "overwrite", "t", true, "overwrite jobs of the same name")
-	packagesDo.Flags().BoolVarP(&do.LocalCompiler, "local-compiler", "z", false, "use a local compiler service; overwrites anything added to compilers flag")
+	packagesDo.Flags().BoolVarP(&do.RemoteCompiler, "remote-compiler", "z", false, "use a remote compiler; if set uses the url specified with the compiler flag.")
```
Comment (benjaminbollen) [inline cmd/pkgs.go]: the functionality of `-z` is changed here; so best remove "z" to avoid unexpected behaviour in existing scripts

**Comment by benjaminbollen** [comment]
woops, approved, but yes, please remove the "-z" other than that it looks good to me

File: cmd/pkgs.go (hunk 2 above)
Comment (VoR0220) [inline cmd/pkgs.go]: fair enough. Will change the flag.

File: cmd/pkgs.go [hunk 3]
```diff
@@ -55,7 +55,7 @@ func addPackagesFlags() {
 	packagesDo.Flags().StringVarP(&do.DefaultFee, "fee", "n", "9999", "default fee to use")
 	packagesDo.Flags().StringVarP(&do.DefaultAmount, "amount", "u", "9999", "default amount to use")
 	packagesDo.Flags().BoolVarP(&do.Overwrite, "overwrite", "t", true, "overwrite jobs of the same name")
-	packagesDo.Flags().BoolVarP(&do.RemoteCompiler, "remote-compiler", "z", false, "use a remote compiler; if set uses the url specified with the compiler flag.")
+	packagesDo.Flags().BoolVarP(&do.RemoteCompiler, "remote-compiler", "r", false, "use a remote compiler; if set uses the url specified with the compiler flag.")
```
Comment (benjaminbollen) [inline cmd/pkgs.go]: Good.

Comment (VoR0220) [inline cmd/pkgs.go]: Merge por favor?

**Comment by zramsay** [comment]
@VoR0220 this broke the `pkgs/pkgs_test.go` => `do.LocalCompiler undefined`

**Pull request merged**